code/modules/research/designs/weapons.dm

```diff
@@ -1,13 +1,13 @@
 /datum/design/nuclear_gun
-	name = "Advanced Energy Gun"
-	desc = "An energy gun with an experimental miniaturized reactor."
+	name = "Advanced Energy Gun Modkit"
+	desc = "Can be used on an energy gun to grant it an experimental miniaturized reactor."
 	id = "nuclear_gun"
 	req_tech = list("combat" = 3, "materials" = 5, "powerstorage" = 3)
 	build_type = PROTOLATHE
-	materials = list(MAT_IRON = 5000, MAT_GLASS = 1000, MAT_URANIUM = 500)
+	materials = list(MAT_IRON = 5000, MAT_GLASS = 1500, MAT_URANIUM = 750)
```

Comment:
I would assume it'd cost less, not more, to make a modkit, considering you're using parts from elsewhere.